en/docs/administer/monitoring-server-health.md

```diff
@@ -154,3 +193,47 @@ To deploy it, copy the `         .jar        ` file to the
 the OSGI bundle and paste it in the
 `         <IS_HOME>/repository/component/dropins/        `
 directory.
+
+Then register the new health checker as below in the 'deployment.toml'
+file if needed.
```

Comment:
Shall we modify this to " as **shown** below"

Reply:
fixed